packages/common-sdk/tests/web3/network/account-requests.test.ts

```diff
@@ -12,47 +12,53 @@ jest.setTimeout(100 * 1000 /* ms */);
 
 describe("account-requests", () => {
   const ctx = createTestContext();
+  // Silence the errors when we evaluate invalid token accounts.
+  beforeEach(() => {
+    jest.spyOn(console, 'error').mockImplementation(() => { });
+  });
 
   beforeAll(async () => {
     await requestAirdrop(ctx);
   });
 
   it("getParsedAccount, ok", async () => {
     const mint = await createNewMint(ctx);
-    const expected = await mint.getMintInfo();
-    const mintInfo = await getParsedAccount(ctx.connection, mint.publicKey, ParsableMintInfo);
-    expectMintInfoEquals(mintInfo!, expected);
+    const expected = await getMint(ctx.connection, mint);
+
+    const mintInfo = await getParsedAccount(ctx.connection, mint, ParsableMintInfo);
+    expectMintEquals(mintInfo!.data, expected);
```

Comment:
When I run ``yarn run test --verbose``, I found the following error. 🤔
```
tests/web3/network/account-requests.test.ts:29:32 - error TS2339: Property 'data' does not exist on type 'Mint'.

    29     expectMintEquals(mintInfo!.data, expected);
```
There were same errors in this file.